script.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+// form issue 해결
+// submit이벤트가 발생하면 브라우저가refresh된다.
+// e.preventDefault()를 통해서 브라우저의 refresh를 막고
+// 새로운 변수 location에 input값 저장해서 textContent로 출력
+const Form = document.querySelector("form");
+const inputBox = document.querySelector("input");
+const waitingList = document.querySelector("#waitingList");
+
+Form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  //const location = inputBox.value;
+  //waitingList.textContent = location;
+});
+
+// input창에 입력
+let itemList = [];
+let finishedItem = [];
+
+let inputButton = document.querySelector("button");
+inputButton.addEventListener("click", addItem);
+
+function addItem() {
+  let item = document.querySelector("#input").value;
+  if (item != null) {
```

Comment:
혹시 이 부분은 공백 입력을 방지하는 코드인가요?

```suggestion
  if (item != '') {
```
그렇다면 `null`이 아닌 `''` 으로 바꿔 주셔야 할 것 같습니다! 
```suggestion
  if (item) {
```
이렇게 작성하셔도 될 거예요!